src/test/kotlin/com/ampnet/web3provider/service/AccountInformationServiceTest.kt

```diff
@@ -45,11 +45,11 @@ class AccountInformationServiceTest : JpaServiceTestBase() {
             accountInformationService.getBalance(testContext.jsonRpcRequest)
             val cache = redisRepository.getCache(RedisEntity.BALANCE.methodName, address + blockParameter)
             val expiryTimeInSec = redisTemplate.getExpire(RedisEntity.BALANCE.methodName)
-            Thread.sleep(RedisEntity.BALANCE.ttlInSec * 1000)
+            Thread.sleep(applicationProperties.redis.ethGetBalanceTtl * 1000)
```

Comment:
Why is this sleep needed here?

Reply:
It checks if the ttl is applied correctly and if cache is cleaned after ttl expires.